docs/configurations.md

```diff
@@ -1,53 +1,70 @@
-| `scalar.db.consensus_commit.serializable_strategy`    | Serializable strategy used for Consensus Commit. Either `EXTRA_READ` or `EXTRA_WRITE` can be specified. If `SNAPSHOT` is specified in the property `scalar.db.consensus_commit.isolation_level`, this is ignored.                                                                                                                                                                                       | `EXTRA_READ`  |
+| `scalar.db.consensus_commit.serializable_strategy`    | Serializable strategy used for Consensus Commit. Either `EXTRA_READ` or `EXTRA_WRITE` can be specified. If `SNAPSHOT` is specified in the property `scalar.db.consensus_commit.isolation_level`, this configuration will be ignored.                                                                                                                                                                                       | `EXTRA_READ`  |
 | `scalar.db.consensus_commit.coordinator.namespace`    | Namespace name of coordinator tables.                                                                                                                                                                                                                                                                                                                                                                   | `coordinator` |
-| `scalar.db.consensus_commit.include_metadata.enabled` | If set to `true`, Get and Scan operations results will contain transaction metadata. To see the transaction metadata columns details for a given table, you can use the `DistributedTransactionAdmin.getTableMetadata()` method which will return the table metadata augmented with the transaction metadata columns. Using this configuration can be useful to investigate transaction related issues. | `false`       |
+| `scalar.db.consensus_commit.include_metadata.enabled` | If set to `true`, Get and Scan operations results will contain transaction metadata. To see the transaction metadata columns details for a given table, you can use the `DistributedTransactionAdmin.getTableMetadata()` method, which will return the table metadata augmented with the transaction metadata columns. Using this configuration can be useful to investigate transaction-related issues. | `false`       |
 
 #### Performance-related configurations
 
-The performance-related configurations for Consensus Commit are as follows:
+The following performance-related configurations are available for Consensus Commit transactions:
```

Comment:
Ditto.

Reply:
I've fixed this to match the previous use.